Pod/Classes/IPDecimalNumberPad.m

```diff
@@ -54,8 +55,11 @@ - (id)initWithCoder:(NSCoder *)aDecoder {
 }
 
 - (void)commonInit {
-    self.buttonClass = [UIButton class];
     self.backgroundColor = [UIColor clearColor];
+    _buttonClass = [IPDecimalNumberPadButton class];
+    _columnDividerImage = [self imageFromAssetBundleNamed:@"verticalLine"];
```

Comment:
Is there a reason to maintain strong references to these images separately? The buttons reference them, correct?